src/main/java/PacMan/Todo.java

```diff
@@ -0,0 +1,25 @@
+package pacman;
+
+/**
+ * Represents a todo task. A <code>Todo</code> object corresponds to
+ * a task's name
+ */
+public class Todo extends Task {
+    Todo(String task) {
+        super(task);
+    }
+
+    /**
+     * Return a <code>String</code> that is readable and writeable by <code>Storage</code>
+     *
+     * @return a <code>String</code> that is readable and writeable by <code>Storage</code>
+     */
+    public String toFile() {
```

Comment:
I like how you used the `Task` superclass here. Good application of inheritance and code reusability.